webpack.icons.config.js

```diff
@@ -0,0 +1,54 @@
+const path = require("path");
+const {UglifyJsPlugin} = require('webpack').optimize;
+
+module.exports = {
+    entry: {
+        "clarity-icons-light.min": "./src/clarity-icons/index.ts",
+        "clarity-icons.min": "./src/clarity-icons/clarity-icons-sfx.ts",
+        "shapes/commerce-shapes": "./src/clarity-icons/shapes/commerce-shapes.ts",
```

Comment:
So for each shape set, are we publishing two `.js` files with the same content; one is just `.js` and `.min.js`?

Reply:
The `.min.js` is the minified version of the `.js` files.